src/SearchResults.js

```diff
@@ -1,44 +1,102 @@
-import React from "react";
+import React, { useState } from "react";
 import { Table } from "reactstrap";
-import moment from "moment";
+import Row from "./row";
 
 const SearchResults = props => {
+  const [active, setActiveClass] = useState(false);
+  function handleClick() {
+    setActiveClass(!active);
+  }
```

Comment:
Is this state being used? I think it might have been left-over from an old version. If it's not being used, could you remove it?

Reply:
@40thieves  I fixed it, I forgot to add onClick  for it